electron_strings.grdp

```diff
@@ -5,6 +5,17 @@
     V8 Proxy Resolver
   </message>
 
+  <!-- Printing Service -->
+  <message name="IDS_UTILITY_PROCESS_PRINTING_SERVICE_NAME" desc="The name of the utility process used for printing conversions.">
```

Comment:
is there a way we could directly rely on `printing_strings.grdp` from chromium (or directly on chrome's generated_resources grit?)

Reply:
I did that for a while, but I ran into some duplication errors, and that also increases the file sizes a bit, since there is a large number of these strings that we don't actually use. I think that if we want to use those girt files, we should go all the way and replace all of what we have here in a separate PR with the chromium equivalent (relying directly on chrome's generated_resources, or just including the relevant bits here).

Reply:
cool, this seems fine for now then.